### PR TITLE
Add UI for library

### PR DESCRIPTION
**Pull request by MesakDuduCoder**
## In this branch I implemented the following

* Create a app.rb file that will serve as the console app entry-point. It has methods that do the following:
   List all books.
   List all people.
   Create a person (teacher or student, not a plain Person).
   Create a book.
   Create a rental.
   List all rentals for a given person id.

* In main.rb I define the entry point, this will be a method called main that is invoked at the end of the file. This method should do the following:
  Present the user with a list of options to perform.
  Lets users choose an option.
  If needed, ask for parameters for the option.
  Have a way to quit the app.
  
* I also include executable file for windows

### CODE REVIEW

File: app.rb [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+      @people << person
+    elsif type == 2
+      puts 'Enter specialization'
+      specialization = gets.chomp
+      person = Teacher.new(age, specialization, name: name)
+      @people << person
+    else
+      puts 'Invalid input'
+    end
+  end
```
Comment (AmaduKamara) [inline app.rb:54]: - [ ] Good job sor on your implementation of creating a person, but however, I highly recommend you stick to the project requirement. Kindly follow the video sneak peek on how to implement this section for creating a person that first asks if a student or a teacher is to be created and based on the option the user enters, that person is being created for them. Currently, your implementation seems confusing in that, you don't know who you are creating from the beginning and only in the end, the option is given. Let's try to have a good user experience of our app by adhering to project guidelines. Please follow the video guide and I'll place screenshots below to compare. 👍🏽

**THE SNEAK PEEK**

Screenshot 2023-08-29 at 4 26 50 PM

**Your Implementation**
Screenshot 2023-08-29 at 4 19 11 PM

**Project Requirement**
Screenshot 2023-08-29 at 4 28 02 PM

**Review by AmaduKamara — CHANGES_REQUESTED** [review]
# Status: Required Changes ♻️ ♻️ ♻️

Hi @MesakDuduCoder,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

✅ You used Gitflow correctly 👍🏽
✅ The Linter checks passed successfully 👍🏽
✅ Good descriptive PR 👍🏽

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: app.rb (hunk 1 above)
Comment (MesakDuduCoder) [reply app.rb]: Thank you @AmaduKamara I will do that

**Review by Strangeal — APPROVED** [review]
Hi @MesakDuduCoder, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 




## Highlights
- Added suggested change ✅ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**